### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by alexugthub**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by alexugthub — COMMENTED** [review]
Look good

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (alexmro) [inline index.html:50]: This was a good changee

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (alexmro) [inline index.html:49]: vsdfsdfasdf

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
-<h2 hidden>Game over</h2>
```
Comment (alexmro) [inline index.html:50]: <h2 hidden>Game over</h2>
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>

Comment (alexmro) [inline index.html:50]: `<h2 hidden>Games over</h2>`

File: index.html (hunk 1 above)
Comment (alexugthub) [inline index.html]:
```suggestion
<h2 hidden>Game over, restart to play again 🧑‍💻 🤖!</h2>
```

**Pull request merged**